eventio/simtel/objects.py

```diff
@@ -597,11 +579,26 @@ def __repr__(self):
 class SimTelEvent(EventIOObject):
     eventio_type = 2010
 
+    def __init__(self, header, parent):
+        super().__init__(header, parent)
+        self.glob_count = header.id
```

Comment:
Not important for this PR .. but I think `glob_count` is the same as `event_id` in other objects, or more precisely I think it is ``shower_id * 100 + reuse_id` ...